ocaml/quicktest/quicktest_fe_systemctl.ml

```diff
@@ -0,0 +1,96 @@
+(*
+ * Copyright (C) Cloud Software Group, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+let home = Sys.getenv_opt "HOME" |> Option.value ~default:"/root"
+
+let dest =
+  let dir =
+    if Unix.getuid () = 0 then
+      "/run/systemd/system"
+    else
+      Filename.concat home ".config/systemd/user"
+  in
+  Filename.concat dir "xapi-envtest@.service"
+
+let get_temp_file prefix suffix =
+  (* /tmp visible to systemd may not match /tmp visible to test
+     when running tests inside containers.
+     Use a path in the current dir instead for testing.
+  *)
+  let name = Filename.temp_file ~temp_dir:(Sys.getcwd ()) prefix suffix in
```

Comment:
`Filename.temp_file ` by default creates a name inside the temp file directory of the system. You are providing a different default here. Would in this case it not be enough to provide "." for the current directory?